Jellyfin.Server/Migrations/JellyfinMigrationService.cs

```diff
@@ -105,6 +106,13 @@ public async Task CheckFirstTimeRunOrMigration(IApplicationPaths appPaths)
             var dbContext = await _dbContextFactory.CreateDbContextAsync().ConfigureAwait(false);
             await using (dbContext.ConfigureAwait(false))
             {
+                var databaseCreator = dbContext.Database.GetService<IDatabaseCreator>() as IRelationalDatabaseCreator
```

Comment:
Why this interface over simply calling EnsureCreated?

Reply:
<img width="2099" height="184" alt="image" src="https://github.com/user-attachments/assets/dcc80820-46e3-4dfe-951c-d5a553d52dfa" />